package.json

```diff
@@ -35,7 +35,7 @@
     "pg-format": "1.0.4",
     "picocolors": "1.0.0",
     "pino": "8.8.0",
-    "postgrator": "3.0.0",
+    "postgrator": "5.0.1",
```

Comment:
Isn't 7.7.1 the latest version?

Reply:
That is ESM. I tried it gave some errors that can not import. So I started to go up iteratively.

Reply:
Sounds good